### PR TITLE
Binairy -> Binary

### LINKED ISSUE #12: Fix naming of BinaryExpressionMutator
The BinaryExpressionMutator is currently called the Bin**airy**ExpressionMutator.

### PR DESCRIPTION
**Pull request by StefanKraai**
Closes #12

### CODE REVIEW

File: src/Stryker.Core/Stryker.Core/Mutators/BinaryExpressionMutator.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public override IEnumerable<Mutation> ApplyMutations(BinaryExpressionSyntax node
                     {
                         OriginalNode = node,
                         ReplacementNode = replacementNode,
-                        DisplayName = "Binairy expression mutation",
+                        DisplayName = "Binary expression mutation",
                         Type = "MathMutator"
```
Comment (simondel) [inline src/Stryker.Core/Stryker.Core/Mutators/BinaryExpressionMutator.cs]: This type is still wrong

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ While developing on Stryker.NET we advise to work in [Visual Studio 2017](https:
 *	Create a new Debug profile
 *	Set `Launch` as `Project` 
 *	Set `WorkingDirectory` as your local installation dir, pointing to a UnitTest project `example: (C:\Repos\stryker-net\src\Stryker.Core\Stryker.Core.UnitTest)`
-*	Run the program with `Stryker.CLI` as the startup project
+*	Run the program with the newly created Debug profile
```
Comment (richardwerkman) [inline CONTRIBUTING.md]: Why did you remove the startup project instruction? Wouldn't this be an extra instruction?
